src/supervisor/apiServerObjects.py

```diff
@@ -36,12 +36,20 @@ def apiServerObjectCount(pc,startTime, endTime, step):
     print("Key APi Server Objects Count")
     objects=['secrets','configmaps','serviceaccounts','clusterrolebindings.rbac.authorization.k8s.io','rolebindings.rbac.authorization.k8s.io',
              'clusterroles.rbac.authorization.k8s.io','roles.rbac.authorization.k8s.io','leases.coordination.k8s.io',
-             'configurationpolicies.policy.open-cluster-management.io','manifestworks.work.open-cluster-management.io',
+             'manifestworks.work.open-cluster-management.io',
              'placements.cluster.open-cluster-management.io','subscriptions.apps.open-cluster-management.io','applications.app.k8s.io',
-             'applications.argoproj.io','applicationsets.argoproj.io']
+             'applications.argoproj.io','applicationsets.argoproj.io','policies.policy.open-cluster-management.io',
+             'configurationpolicies.policy.open-cluster-management.io','certificatepolicies.policy.open-cluster-management.io',
+             'iampolicies.policy.open-cluster-management.io','operatorpolicies.policy.open-cluster-management.io',
```

Comment:
We are removing iampolicies in the next release.  If you think that will be a problem with this list then it could be removed.  If a missing CRD doesn't cause problems then you could keep it.  Either way we are not going to be very interested in that particular resource.

Reply:
Removed for simplicity.